structure/struct.py

```diff
@@ -604,8 +604,12 @@ def get_league_matches(self):
                 match = Match(
                     match_id, radiant_team_id, dire_team_id, self.league_id, radiant_win
                 )
-                match.get_match_data()
-                self.add_match(match)
+                try:
+                    match.get_match_data()
+                    self.add_match(match)
+                except (TypeError, KeyError):
```

Comment:
It is possible to cover it somehow by some tests?

Reply:
it is a floating thing, somitimse opendota return broken data, which not contains all needed info, it occures once pre 500 matches, so i added it to not braek the runtime